src/pages/home/report/ReportActionsView.js

```diff
@@ -458,7 +458,7 @@ export default compose(
             key: ONYXKEYS.IS_LOADING_REPORT_DATA,
         },
         isLoadingReportActions: {
-            key: ONYXKEYS.IS_LOADING_REPORT_ACTIONS,
+            key: ONYXKEYS.COLLECTION.IS_LOADING_REPORT_ACTIONS,
```

Comment:
Couple other things here...

1. If we're not using `ONYXKEYS.IS_LOADING_REPORT_ACTIONS` then we have to remove the key
2. This is going to return an object because we're using a collection key. If we want to figure out if this specific report is loading then we need a changed like this:

```suggestion
            key: ({reportID}) => `${ONYXKEYS.COLLECTION.IS_LOADING_REPORT_ACTIONS}${reportID}`,
```

Reply:
thanks I would have taken some time to figure that one out